files/ja/web/api/canvaspattern/settransform/index.md

```diff
@@ -41,7 +41,7 @@ var svg1 = document.getElementById("svg1");
 var matrix = svg1.createSVGMatrix();
 
 var img = new Image();
-img.src = 'https://mdn.mozillademos.org/files/222/Canvas_createpattern.png';
+img.src = 'canvas_createpattern.png';
```

Comment:
The link in this URL is broken, because it is in an example code.
Can you fix a link to work correctly, for example, linking to the English version?

Reply:
Unfortunately, this is a known problem: https://github.com/mdn/yari/issues/5652

I'm planning on working on this issue next week.